src/TcxWriter.cpp

```diff
@@ -77,21 +77,8 @@ void TcxWriter::onWorkout(const WorkoutInfo &i)  {
     writer.startElement("Activity");
     writer. writeAttribute("Sport", sport);
     writer. writeElement("Id", i.start_time.format());
-    writer. startElement("Lap");
-    writer. writeAttribute("StartTime", i.start_time.format());
-    writer. writeElement("TotalTimeSeconds", "0");
-    writer. writeElement("DistanceMeters", "0");
-      {
-        std::ostringstream ss;
-        ss<< (int)(i.calories/100);
-        writer.writeElement("Calories", ss.str());
```

Comment:
Andrew,

One option is to put the total calories in the calories field of the last lap. Or go for an extension to the schema.
Not sure which will work best.
My experience with RunKeeper is that if data is omitted (laptimes, calories, etc) they calculate their own data based on the waypoints. Would be nice though if RunKeeper can use the data we provide in the TCX file.